### PR TITLE
Added Feature Matrix for Intro.md

### PR DESCRIPTION
**Pull request by Tsquash**
From issue #179 Still need to decide what goes in the top-left most box because feature is already used as the header. Also need to come up with a sentence to go above the table that relates to the table featuring zmk and qmk. More features could also be added but this is a good start.

### CODE REVIEW

File: docs/docs/intro.md [hunk 1]
```diff
@@ -13,31 +13,30 @@ feature.
...
-
-- One Shot Keys
-- Combo keys
-- Macros
-- Complete split support (encoders and RGB are not supported on the 'peripheral' side)
-- Battery reporting
-- Low power sleep states
-- Low power mode (to toggle LEDs and screen off)
-- Shell over BLE
+At this point, ZMK is still missing many features compared to more developed firmwares such as QMK. Here is a table comparing supported ZMK and QMK features:
```
Comment (Tsquash) [inline docs/docs/intro.md:16]: fix typo from firmwares to firmware

File: docs/docs/intro.md [hunk 2]
```diff
@@ -13,31 +13,30 @@ feature.
...
+ 
+|                                                                                             |    ZMK    |     QMK     |
+|----------------------------------------------------------------------------------------------------------|:---------:|:-----------:|
```
Comment (Tsquash) [inline docs/docs/intro.md:19]: Something needs to go in this top left box, not sure what though. Features is already used as a header above this table.

File: docs/docs/intro.md [hunk 3]
```diff
@@ -13,32 +13,32 @@ feature.
...
+| Combo Keys                                                                                               |  In Dev         |      ✅      |
+| Macros                                                                                                   |  In Dev         |      ✅      |
+| Mouse Keys                                                                                               |           |      ✅      |
+| Battery Reporting                                                                                        | In Dev          |             |
+| Low Power Sleep States                                                                                   |  In Dev         |             |
+| Low Power Mode (VCC Shutoff)                                                                             | In Dev          |             |
+| Shell over BLE                                                                                           |           |             |
+| Realtime Keymap Updating                                                                                 |     In Dev    |      ✅       |
```
Comment (petejohanson) [inline docs/docs/intro.md:39]: This really is at best "Planned":

```suggestion
| Realtime Keymap Updating                                                                                 |     Planned    |      ✅       |
```

File: docs/docs/intro.md [hunk 4]
```diff
@@ -13,32 +13,32 @@ feature.
...
+| USB Connectivity                                                                                         |     ✅     |      ✅      |
+| Low Active Power Usage                                                                                   |     ✅     |             |
+| Split Keyboard Support                                                                                   |     ✅     |      ✅      |
+| [Keymaps and Layers](behavior/layers)                                                                    |     ✅     |      ✅      |
+| [Hold-tap](behavior/hold-tap) (which includes [mod-tap](behavior/mod-tap), [layer-tap](behavior/layers)) |     ✅     |      ✅      |
+| [Basic HID over USB](behavior/key-press)                                                                 |     ✅     |      ✅      |
+| [Basic consumer (media) keycodes](behavior/key-press#consumer-key-press)                                 |     ✅     |      ✅      |
+| [Encoders](feature/encoders)                                                                             |     ✅     |      ✅      |
```
Comment (petejohanson) [inline docs/docs/intro.md:28]: This really should have a footnote that warns that encoders on split peripheral sides are not yet supported.

File: docs/docs/intro.md [hunk 5]
```diff
@@ -13,32 +13,32 @@ feature.
...
+| Basic [OLED display support](feature/displays)                                                           |     ✅     |      ✅      |
```
Comment (petejohanson) [inline docs/docs/intro.md:29]: Honestly, at this point I would call this not even "Basic" but "Proof of Concept"

File: docs/docs/intro.md [hunk 6]
```diff
@@ -13,32 +13,32 @@ feature.
...
-- Battery reporting
-- Low power sleep states
-- Low power mode (to toggle LEDs and screen off)
-- Shell over BLE
-
+At this point, ZMK is still missing many features compared to more developed firmware such as QMK. Here is a table comparing supported ZMK and QMK features:
+ 
+|   **Feature**                                                                                          |   &nbsp  ZMK    &nbsp |    &nbsp  QMK   &nbsp   |
+|----------------------------------------------------------------------------------------------------------|:---------:|:-----------:|
+| Wireless Connectivity via BLE                                                                            |     ✅     |             |
```
Comment (petejohanson) [inline docs/docs/intro.md:20]: We should call out "Multi Device BLE Support", not that we support multiple profiles.

File: docs/docs/intro.md [hunk 7]
```diff
@@ -13,32 +13,32 @@ feature.
...
+| USB Connectivity                                                                                         |     ✅     |      ✅      |
+| Low Active Power Usage                                                                                   |     ✅     |             |
+| Split Keyboard Support                                                                                   |     ✅     |      ✅      |
+| [Keymaps and Layers](behavior/layers)                                                                    |     ✅     |      ✅      |
+| [Hold-tap](behavior/hold-tap) (which includes [mod-tap](behavior/mod-tap), [layer-tap](behavior/layers)) |     ✅     |      ✅      |
+| [Basic HID over USB](behavior/key-press)                                                                 |     ✅     |      ✅      |
```
Comment (petejohanson) [inline docs/docs/intro.md:26]: This should be reworded, and the USB part of this should be removed.. Perhaps "Basic Keycodes" is enough?

File: docs/docs/intro.md [hunk 8]
```diff
@@ -13,32 +13,32 @@ feature.
...
+ 
+|   **Feature**                                                                                          |   &nbsp  ZMK    &nbsp |    &nbsp  QMK   &nbsp   |
+|----------------------------------------------------------------------------------------------------------|:---------:|:-----------:|
+| Wireless Connectivity via BLE                                                                            |     ✅     |             |
+| USB Connectivity                                                                                         |     ✅     |      ✅      |
+| Low Active Power Usage                                                                                   |     ✅     |             |
+| Split Keyboard Support                                                                                   |     ✅     |      ✅      |
+| [Keymaps and Layers](behavior/layers)                                                                    |     ✅     |      ✅      |
+| [Hold-tap](behavior/hold-tap) (which includes [mod-tap](behavior/mod-tap), [layer-tap](behavior/layers)) |     ✅     |      ✅      |
```
Comment (petejohanson) [inline docs/docs/intro.md:25]: You should be able to link to the specific anchor for layer-tap here.

File: docs/docs/intro.md [hunk 9]
```diff
@@ -13,32 +13,32 @@ feature.
...
-- Macros
-- Complete split support (encoders and RGB are not supported on the 'peripheral' side)
-- Battery reporting
-- Low power sleep states
-- Low power mode (to toggle LEDs and screen off)
-- Shell over BLE
-
+At this point, ZMK is still missing many features compared to more developed firmware such as QMK. Here is a table comparing supported ZMK and QMK features:
+ 
+|   **Feature**                                                                                          |   &nbsp  ZMK    &nbsp |    &nbsp  QMK   &nbsp   |
```
Comment (petejohanson) [inline docs/docs/intro.md:18]: Can we also call out the user repo feature? This is a big deal, not forking, nor having to compile locally. Maybe called "User Configs/Keymaps with Build Service"?

**Review by petejohanson — COMMENTED** [review]
Thanaks for working on this. A few thoughts.

File: docs/docs/intro.md [hunk 10]
```diff
@@ -13,31 +13,34 @@ feature.
...
-
-- One Shot Keys
-- Combo keys
-- Macros
-- Complete split support (encoders and RGB are not supported on the 'peripheral' side)
-- Battery reporting
-- Low power sleep states
-- Low power mode (to toggle LEDs and screen off)
-- Shell over BLE
+At this point, ZMK is still missing many features compared to more developed firmware such as QMK. Here is a table comparing supported ZMK and QMK features:
```
Comment (petejohanson) [inline docs/docs/intro.md:16]: Perhaps instead of "more developed", we could say "advanced" or even just "popular"?

**Comment by innovaker** [comment]
May I suggest that we don't just compare against QMK?  It might inadvertently give the impression that ZMK is intended to supplant it - which it isn't.   Perhaps a column for BlueMicro too?

( see http://bluemicro.jpconstantineau.com/#features.html )

**Comment by innovaker** [comment]
If we wanted to make it more visual, an alternative to "In Dev" might be 🔨 or 🔧.  Any thoughts?

**Comment by innovaker** [comment]
For large pieces of work such as a behavior that are "In Dev", we might want to link to the parent GitHub issue so users can easily find out more and track progress?

File: docs/docs/intro.md [hunk 11]
```diff
@@ -9,11 +9,12 @@ firmware built on the [Zephyr™ Project](https://zephyrproject.org/) Real Time
 
 ## Features
 
-At this point, ZMK is still missing many features compared to popular firmware such as QMK. Here is a table comparing supported ZMK and QMK features:
+At this point, ZMK is still missing many features compared to popular firmware such as QMK. Here is a table comparing supported ZMK, BlueMicro, and QMK features:
```
Comment (innovaker) [inline docs/docs/intro.md:12]: Now that we've added a BM column, perhaps this is more appropriate:
`ZMK is currently missing some features found in other popular firmware.  This table compares the features supported by ZMK, BlueMicro and QMK:`

**Comment by innovaker** [comment]
I can draw up a specification for `Mouse Keys` if we want it `Planned`.

**Comment by innovaker** [comment]
Also do we want to be explicit on stuff  we'll likely never support - with a ✖?  Just a thought.

File: docs/docs/intro.md [hunk 12]
```diff
@@ -5,39 +5,40 @@ sidebar_label: Introduction
...
+| Low Latency BLE Support                                                                                |     ✅      |       ✅    |             | 
+| Multi-Device BLE Support                                                                               |     ✅      |              |             |
+| USB Connectivity                                                                                       |     ✅      |              |      ✅     |
+| User Configuration Repositories                                                                        |     ✅      |              |             |
+| Split Keyboard Support                                                                                 |     ✅      |      ✅      |     ✅     |   
+| [Keymaps and Layers](behavior/layers)                                                                  |     ✅      |      ✅      |     ✅     |
+| [Hold-Tap](behavior/hold-tap) (Which Includes [Mod-Tap](behavior/mod-tap) and [Layer-Tap](behavior/layers/#layer-tap)) |     ✅      |     ✅    |    ✅   |
```
Comment (innovaker) [inline docs/docs/intro.md:22]: `which includes`?

**Comment by innovaker** [comment]
Thanks @Tsquash!  I'll take another look over the coming days.

**Review by petejohanson — APPROVED** [review]
LGTM. I'll let you @innovaker weigh in once more before we merge.

**Comment by innovaker** [comment]
As I mentioned to @Tsquash over DMs today, I've been sick and was intending to follow this up after.

**Pull request merged**